docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile

```diff
@@ -20,7 +20,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install initramfs-tools libnuma-de
         wget ${ROCM_INSTALLER_REPO} && \
         sudo apt-get install -y ./${ROCM_INSTALLER_PACKAGE} && \
         sudo apt-get update -y && \
-        sudo amdgpu-install -y --usecase=graphics,rocm --no-32
+        sudo amdgpu-install -y --usecase=rocm
```

Comment:
do we work with ffmpeg HW acceleration when we remove this usecase

Reply:
We do not have the graphics usecase going forward from rocm6.2. @kiritigowda please correct me if I am wrong here. We need to install the dependencies like we do for rocDecode I think